python/ray/serve/scripts.py

```diff
@@ -427,7 +341,6 @@ def status(address: str):
 )
 @click.option("--yes", "-y", is_flag=True, help="Bypass confirmation prompt.")
 def delete(address: str, yes: bool):
```

Comment:
Should this be changed to `shutdown` now or after Serve controller changes are made?